build/scripts/build-utils.ps1

```diff
@@ -156,7 +164,7 @@ function Ensure-SdkInPathAndData() {
     if ($dotnetDir -ne $null) { 
         $sdkPath = Join-Path $dotnetDir "sdk\$sdkVersion"
```

Comment:
Does Powershell on Unix automatically do the right thing for the path separate character here? Does this also work for network paths?

Reply:
Powershell on linux accepts both `/` and `\`. (Windows powershell does too)

> Does this also work for network paths?

What do you mean?

Reply:
Just that there were some APIs that didn't necessarily play nicely with the differing PathSeparatorCharacter last time I checked (intermixing `\` and `/` would cause crashes) and some that didn't work well if the folder was actually on a network path.

The issues may have since been fixed.